### PR TITLE
[TERMAPI-218] Add the ability to control the screensaver mode

### PR DESCRIPTION
**Pull request by denys-marichev-sumup**
[TERMAPI-218](https://sumupteam.atlassian.net/browse/TERMAPI-218): Add the ability to control the screensaver mode

[TERMAPI-218]: https://sumupteam.atlassian.net/browse/TERMAPI-218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: terminal_api_flows/__init__.py [hunk 1]
```diff
@@ -65,6 +65,36 @@ def skip_screen():
...
+    )
+
+    if res.status == 200 and json_data["status"] == "SUCCESS":
+        print_outcome("SUCCESS", res.status, json_data)
+    else:
+        print_outcome("FAILED", res.status, json_data)
+
+
+def turn_on_screensaver() -> None:
+    print("Turning ON screensaver:")
```
Comment (chad-lung-fivestars) [inline terminal_api_flows/__init__.py]: Let's change the wording of this to: `Activating screensaver on customer terminal`

File: terminal_api_flows/__init__.py [hunk 2]
```diff
@@ -65,6 +65,36 @@ def skip_screen():
...
+    _toggle_screensaver(screensaver=True)
+
+
+def turn_off_screensaver() -> None:
+    print("Turning OFF screensaver:")
+    _toggle_screensaver(screensaver=False)
```
Comment (chad-lung-fivestars) [inline terminal_api_flows/__init__.py:95]: Let's change the wording of this to: `Deactivating screensaver on customer terminal`

**Review by chad-lung-fivestars — CHANGES_REQUESTED** [review]
A couple comments otherwise this looks good.

File: terminal_api_flows/__init__.py (hunk 2 above)
Comment (denys-marichev-sumup) [reply terminal_api_flows/__init__.py:95]: Fixed it in cb6f630 :heavy_check_mark:

File: terminal_api_flows/__init__.py (hunk 1 above)
Comment (denys-marichev-sumup) [reply terminal_api_flows/__init__.py]: Fixed it in cb6f630 :heavy_check_mark:

**Review by chad-lung-fivestars — APPROVED** [review]
Looks good, thanks. 👍

**Pull request merged**